vanilla-framework/vanilla-framework
# Update Vanilla Docs to 1.7.0

## Done

- Update docs to 1.7.0 now that it's released

## QA

- Pull code, `cd docs && ./run`
- Open http://0.0.0.0:8104/
- Check it looks okay
